src/components/global/Playground/index.tsx

```diff
@@ -372,6 +372,7 @@ export default function Playground({
       // using outerText will preserve line breaks for formatting in Stackblitz editor
       codeBlock = codeRef.current.querySelector('code').outerText;
     } else {
+      codeBlock = codeRef.current.querySelector('code').outerText;
```

Comment:
Can we refactor the code now that we are doing the same operation in both cases?

e.g.:
```ts
// using outerText will preserve line breaks for formatting in Stackblitz editor
const codeBlock = codeRef.current.querySelector('code').outerText;
if (hasUsageTargetOptions) {
  codeBlock = codeRef.current.querySelector('code').outerText;
  editorOptions.files = Object.keys(codeSnippets[usageTarget])
    .map((fileName) => ({
      [fileName]: hostRef.current!.querySelector<HTMLElement>(`#${getCodeSnippetId(usageTarget, fileName)} code`)
        .outerText,
    }))
    .reduce((acc, curr) => ({ ...acc, ...curr }), {});
  editorOptions.dependencies = (code[usageTarget] as UsageTargetOptions).dependencies;
}
```

Reply:
oh yeah, good catch!